### PR TITLE
CIS AWS - Add rule 2.2.1 (EBS encryption)

### PR DESCRIPTION
**Pull request by uri-weisman**
Add rule 2.2.1 (EBS encryption)

### Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

> **Note** Make sure to bump cloudbeat [`version/policy.go`](https://github.com/elastic/cloudbeat/blob/main/version/policy.go#L20) after merging this PR and drafting a new release.

### CODE REVIEW

File: bundle/compliance/policy/aws_ec2/data_adapter.rego [hunk 1]
```diff
@@ -22,7 +26,9 @@ security_groups_ip_permissions = entries {
 
 is_default_security_group {
 	input.resource.GroupName == "default"
-} else = false
+} else = false {
+	true
+}
```
Comment (oren-zohar) [inline bundle/compliance/policy/aws_ec2/data_adapter.rego:31]: you need to update your local OPA version and re-run the pre-commit hooks

File: bundle/compliance/cis_aws/rules/cis_2_2_1/rule.rego [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(rule_evaluation),
+		input.resource,
+	)
+}
+
+rule_evaluation {
+	input.resource.enabled == true
```
Comment (oren-zohar) [inline bundle/compliance/cis_aws/rules/cis_2_2_1/rule.rego:21]: `enabled`? shouldn't it be something like `EbsEncryptionByDefault`?

**Review by oren-zohar — CHANGES_REQUESTED** [review]

File: bundle/compliance/cis_aws/test_data.rego [hunk 3]
```diff
@@ -220,6 +220,12 @@ generate_vpc_resource(flow_logs) = {
 	"subType": "aws-vpc",
 }
 
+generate_ebs_encryption_resource(encryption_enabled) = {
+	"resource": {"enabled": encryption_enabled},
+	"type": "ec2",
```
Comment (olegsu) [inline bundle/compliance/cis_aws/test_data.rego:225]: the type is actually `cloud-compute`

**Review by olegsu — APPROVED** [review]

File: bundle/compliance/cis_aws/rules/cis_2_2_1/rule.rego (hunk 2 above)
Comment (uri-weisman) [reply bundle/compliance/cis_aws/rules/cis_2_2_1/rule.rego:21]: Based on
https://github.com/elastic/cloudbeat/blob/main/resources/providers/awslib/ec2/provider.go#L134

Comment (oren-zohar) [reply bundle/compliance/cis_aws/rules/cis_2_2_1/rule.rego:21]: ok please make it more informative next time if possible :pray:

**Review by oren-zohar — APPROVED** [review]

**Pull request merged**